src/dotnet/APIView/APIViewWeb/generated.txt

```diff
@@ -0,0 +1,61 @@
+namespace APIViewWeb {
```

Comment:
This file shouldn't be in the version control.

Reply:
That's a good point. How should I store it in the meantime, before I move on from copying uploaded assemblies locally?

Reply:
Let's switch to uploading stream as part from this PR, it shouldn't be too hard.

Reply:
```
var stream = new MemoryStream(Encoding.UTF8.GetBytes(text));
```